src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -16,23 +16,31 @@
 
 package com.microsoft.hyperspace.index
 
-import org.apache.spark.sql.SparkSession
-import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
 import org.apache.spark.sql.types.{DataType, StructType}
 
+import com.microsoft.hyperspace.HyperspaceException
 import com.microsoft.hyperspace.actions.Constants
-import com.microsoft.hyperspace.index.serde.LogicalPlanSerDeUtils
+import com.microsoft.hyperspace.index.Content.Directory
 
 // IndexLogEntry-specific fingerprint to be temporarily used where fingerprint is not defined.
 case class NoOpFingerprint() {
   val kind: String = "NoOp"
   val properties: Map[String, String] = Map()
 }
 
+case class RelationMetadataEntry(
+    rootPaths: Seq[String],
+    files: Seq[String],
```

Comment:
I couldn't find this `files` variable used. Is it used somewhere or it can be removed?